src/components/node-list/node-list-items.js

```diff
@@ -117,8 +116,8 @@ export const getFilteredTagItems = createSelector(
       faded: false,
       visible: true,
       disabled: false,
-      unset: typeof tagsEnabled[tag.id] === 'undefined',
-      checked: tagsEnabled[tag.id] === true,
+      unset: !tag.enabled,
```

Comment:
Originally the `unset` state was specifically different from `true` and `false` for the UI pattern to work, but maybe things have changed since then.

Looks to work from my testing, so if you're confident this is no longer needed then I'd say consider removing the `unset` flag here and just use `!checked` where it was used before - though if that's a lot of work no problem.

(if it turns out that doesn't work, then maybe it still needs to be distinguished?)

Reply:
Yeah, I have a suspicion that that should be removed. If you don't mind, I might make a new ticket and leave that for a separate PR? It seems tricky.